lib/cylc/conditional_simplifier.py

```diff
@@ -100,31 +101,40 @@ def nest_by_oper( self, nest_me, oper ):
         else:
             return nest_me
 
-    def clean_expr( self, nested_list, criteria ):
-        """Return a list with entries specified by 'critria' removed"""
+    def clean_expr( self, nested_list, criterion ):
+        """Return a list with entries specified by 'criterion' removed"""
```

Comment:
As called from `get_cleaned()` the (formerly) `criteria` argument is a single value, not a list (but can the values be lists?).